feat: Updating the grafeas protos to include compliance and intoto attestation protos

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/82b4f6bf-316e-4bc0-948d-563555103416/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

PiperOrigin-RevId: 414718329
Source-Link: https://github.com/googleapis/googleapis/commit/297bfd86f13acc8e06251bc0b23207f91bcba081
PiperOrigin-RevId: 408420890
Source-Link: https://github.com/googleapis/googleapis/commit/2921f9fb3bfbd16f6b2da0104373e2b47a80a65e
PiperOrigin-RevId: 407119231
Source-Link: https://github.com/googleapis/googleapis/commit/381ab78414f71d6f9476c3044c212e13d762e822

chanseokoh commented:
Ticked the checkbox to have authsynth re-generate this. May take up to 24 hours?

yoshi-automation committed:
feat: Updating the grafeas protos to include compliance and intoto attestation protos

PiperOrigin-RevId: 407119231

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Nov 2 10:32:10 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 381ab78414f71d6f9476c3044c212e13d762e822
Source-Link: https://github.com/googleapis/googleapis/commit/381ab78414f71d6f9476c3044c212e13d762e822

yoshi-automation committed:
chore: update Java and Python dependencies

PiperOrigin-RevId: 408420890

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Nov 8 13:03:45 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 2921f9fb3bfbd16f6b2da0104373e2b47a80a65e
Source-Link: https://github.com/googleapis/googleapis/commit/2921f9fb3bfbd16f6b2da0104373e2b47a80a65e

chanseokoh commented:
@googleapis/cicd any idea why it fails only on Windows?

chanseokoh commented:
The failure is likely due to having the same file name but with different capitalization. For example, we have
```
src/main/java/io/grafeas/v1/IntotoProvenance.java
src/main/java/io/grafeas/v1/InTotoProvenance.java
src/main/java/io/grafeas/v1/IntotoStatement.java
src/main/java/io/grafeas/v1/InTotoStatement.java
```
Leaving @googleapis/cicd to fix.

Neenu1995 committed:
Merge branch 'main' into autosynth-googleapis

Neenu1995 commented:
Windows check failure due to the issue mentioned above. The proto files need to be changed to generate the java files properly before merging.

Neenu1995 commented:
Closing this to regenerate a new PR
